RealEstate/urls.py

```diff
@@ -19,5 +19,8 @@
 
 urlpatterns = [
     url(r'^admin/', include(admin.site.urls)),
-    url(r'^$', include('RealEstate.apps.core.urls')),
+    url(r'^$', include('RealEstate.apps.core.urls'), name='home'),
+    url(r'^login/$', 'django.contrib.auth.views.login', name='auth_login'),
+    url(r'^logout/$', 'django.contrib.auth.views.logout', {'next_page': '/'}, name='auth_logout'),
+    url(r'^logout/(?P<next_page>)/$', 'django.contrib.auth.views.logout', name='auth_logout_next'),
```

Comment:
Django has a login_then_logout view which redirects to the login page after logging out, which sounds like the behavior we want.